go/cmd/vtctldclient/command/root.go

```diff
@@ -54,14 +68,36 @@ var (
 	commandCtx    context.Context
 	commandCancel func()
 
+	// Register functions to be called when the command completes.
+	onTerm = []func(){}
+
+	// Register our nil tmclient grpc handler only one time.
+	// This is primarily for tests where we execute the root
+	// command multiple times.
+	once = sync.Once{}
+
 	server        string
 	actionTimeout time.Duration
 	compactOutput bool
 
+	topoOptions = struct {
+		implementation        string
+		globalServerAddresses []string
+		globalRoot            string
+	}{ // Set defaults
+		implementation:        "etcd2",
+		globalServerAddresses: []string{"localhost:2379"},
+		globalRoot:            "/vitess/global",
+	}
+
 	// Root is the main entrypoint to the vtctldclient CLI.
 	Root = &cobra.Command{
 		Use:   "vtctldclient",
 		Short: "Executes a cluster management command on the remote vtctld server.",
+		Long: fmt.Sprintf(`If there are no running vtctld servers -- for example when bootstrapping
```

Comment:
ah, we need to keep the `Short` as the first sentence here (imo)

Reply:
Good catch! I didn't pay too much attention to the help text output and didn't realize the short was then not shown.